frontend/benefit/applicant/src/backend-api/BackendAPIProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import Axios from 'axios';
+import React from 'react';
+
+import { getBackendDomain } from './backend-api';
+import BackendAPIContext from './BackendAPIContext';
+
+const BackendAPIProvider: React.FC = ({ children }) => {
```

Comment:
This is copy-paste so I was thinking about if the component would be moved to shared folder. The context would be project-speficified and given in `value` prop, and use it like this in `_app.tsx`:
```
import BackendAPIProvider from 'shared/..../BackendAPIProvider'
import BackendAPIContext from 'benefit/..../BackendAPIContext';

<BackendAPIProvider value={BackendAPIContext}>
   ....
</BackendAPIProvider>
```

Reply:
Done